packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.js

```diff
@@ -340,4 +342,90 @@ describe("BirthdayPicker", () => {
             expect(onChange).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe("labels", () => {
+        const translatedLabels: $Shape<Labels> = {
+            month: "Mes",
+            day: "Día",
+            year: "Año",
+            errorMessage: "Por favor ingrese una fecha valida.",
+        };
+
+        beforeEach(() => {
+            jest.useRealTimers();
+            DateMock.advanceTo(today);
+        });
+
+        it.each([defaultLabels.month, defaultLabels.day, defaultLabels.year])(
+            "renders the placeholder as %s",
+            (label) => {
+                // Arrange
+
+                // Act
+                render(<BirthdayPicker onChange={() => {}} />);
+
+                // Assert
+                expect(screen.getByText(label)).toBeInTheDocument();
+            },
+        );
+
+        it.each([
+            translatedLabels.month,
+            translatedLabels.day,
+            translatedLabels.year,
+        ])("renders the translated placeholder as %s", (translatedLabel) => {
+            // Arrange
+
+            // Act
+            render(
+                <BirthdayPicker
+                    onChange={() => {}}
+                    labels={translatedLabels}
+                />,
+            );
+
+            // Assert
+            expect(screen.getByText(translatedLabel)).toBeInTheDocument();
+        });
+
+        it("merges correctly the labels", () => {
+            // Arrange
+
+            // Only passing some of the labels to verify if the others are
+            // merged correctly.
+            const partialLabels: $Shape<Labels> = {
+                month: "Mes",
+                year: "Año",
+            };
```

Comment:
This doesn't seems possible given all fields are required in `Labels`.

Reply:
This is the expected scenario, so I'll probably change the fields as optional.

Reply:
Why would we want some of the fields to be optional?